sdk/containerregistry/container-registry/swagger/containerregistry.json

```diff
@@ -692,19 +652,15 @@
         }
       },
       "patch": {
-        "tags": [
-          "Registry"
-        ],
+        "tags": ["Registry"],
         "description": "Update the attribute identified by `name` where `reference` is the name of the repository.",
         "operationId": "ContainerRegistry_UpdateRepositoryAttributes",
```

Comment:
Would it be possible to move this to `ContainerRepositoryClient` and rename the method to `SetProperties()`?  (If "SetProperties" is .NET specific and other languages use "UpdateProperties" I'm fine putting that in a .NET-specific transform as well.

Reply:
Done. We could still use a different name in convenience layer.